jenkins.sh

```diff
@@ -1,17 +1,13 @@
 #!/bin/bash -eux
 
-shutdown="$(realpath shutdown.sh)"
-trap "sh $shutdown" EXIT
+pwd=$PWD
+function cleanup {
+  docker-compose down -v
+  rm -r verify-matching-service-test-tool-* matching-service-test-tool-*.zip
+}
+trap cleanup EXIT
 
-# Start example local matching service with database migration
-./startup.sh -x clean
+./download-test-tool-from-artifactory.sh
 
-sleep 10 # to give the above application time to start
-
-# Unpack, configure and start test tool
-pushd ../verify-matching-service-test-tool
-    unzip verify-matching-service-test-tool-*.zip
-    cd verify-matching-service-test-tool-*
-
-    ./bin/verify-matching-service-test-tool -e ../../verify-local-matching-service-example/examples -c ../../verify-local-matching-service-example/verify-matching-service-test-tool.yml
-popd
+docker-compose build app
+docker-compose run app ./run-test-tool-scenarios.sh
```

Comment:
where is the database container started?

Reply:
docker-compose starts the application and the database. There is a docker-compose.yml file where we specify application depends on the postgres database. It starts the database container before it starts the application